### PR TITLE
MVVM structure and show toast functionality

### PR DESCRIPTION
**Pull request by florenciaferraiolo**
## Description

MVVM classes and functionality to show toast on clicking the button added

## Evidences

Captura de Pantalla 2022-04-01 a la(s) 15 59 28
Captura de Pantalla 2022-04-01 a la(s) 17 17 59

## Tested on

Device: Nexus S

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -37,10 +37,16 @@ android {
...
+
+    buildFeatures {
+        viewBinding = true
+    }
+
 }
 
 dependencies {
 
+    implementation 'androidx.databinding:library:3.2.0-alpha11'
```
Comment (SartiMau) [inline app/build.gradle]: why do you need this lib? if it is needed please do not use alpha versions

File: app/build.gradle [hunk 2]
```diff
@@ -49,6 +55,7 @@ dependencies {
     implementation generalDependencies.appcompat
     implementation generalDependencies.material
     implementation generalDependencies.constraintlayout
+    implementation 'androidx.lifecycle:lifecycle-viewmodel'
```
Comment (SartiMau) [inline app/build.gradle]: if you will add a dependency, please add it in the general dependencies file

File: app/src/main/java/com/mobile/tandil/kotlinbaseproject/MainActivity.kt [hunk 3]
```diff
@@ -1,11 +1,38 @@
...
 class MainActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainBinding
+    private val viewModel = MainActivityViewModel()
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        val view = binding.root
```
Comment (SartiMau) [inline app/src/main/java/com/mobile/tandil/kotlinbaseproject/MainActivity.kt]: this variable is not needed, please delete it

File: app/src/main/java/com/mobile/tandil/kotlinbaseproject/mvvm/model/Model.kt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+package com.mobile.tandil.kotlinbaseproject.mvvm.model
+
+class Model
```
Comment (SartiMau) [inline app/src/main/java/com/mobile/tandil/kotlinbaseproject/mvvm/model/Model.kt]: add a TODO comment explaining why this class is empty

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/mobile/tandil/kotlinbaseproject/mvvm/model/Model.kt (hunk 4 above)
Comment (florenciaferraiolo) [reply app/src/main/java/com/mobile/tandil/kotlinbaseproject/mvvm/model/Model.kt]: done

File: app/src/main/java/com/mobile/tandil/kotlinbaseproject/MainActivity.kt (hunk 3 above)
Comment (florenciaferraiolo) [reply app/src/main/java/com/mobile/tandil/kotlinbaseproject/MainActivity.kt]: done

File: app/build.gradle (hunk 1 above)
Comment (florenciaferraiolo) [reply app/build.gradle]: done

File: app/build.gradle (hunk 2 above)
Comment (florenciaferraiolo) [reply app/build.gradle]: done

**Review by SartiMau — APPROVED** [review]

**Review by mariano98montero — APPROVED** [review]

**Pull request merged**